xcodeproj/internal/templates/bazel_build.sh

```diff
@@ -103,6 +103,13 @@ if [[ -n "${TOOLCHAINS-}" ]]; then
   fi
 fi
 
+# We unset Metal toolchains since Xcode 26+ complains when that's passed as action_env
+# Older Xcode versions don't seem to populate this variable with Metal toolchain
+# This is why there is no version check
+if [[ "${toolchain:-}" == *"com.apple.dt.toolchain.Metal"* ]]; then
+  unset toolchain
```

Comment:
Would it be better to remove the Metal toolchain instead? Unsetting it will prevent custom toolchains from working if there are multiple toolchains set, won't it?